metainfo.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/dists/org.scummvm.scummvm.metainfo.xml b/dists/org.scummvm.scummvm.metainfo.xml
+index c8da017405b..a499cf8718a 100644
+--- a/dists/org.scummvm.scummvm.metainfo.xml
++++ b/dists/org.scummvm.scummvm.metainfo.xml
+@@ -1,7 +1,7 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <!-- Copyright 2020-2023 The ScummVM Team -->
+ <component type="desktop-application">
+-  <id>org.scummvm.scummvm</id>
++  <id>org.scummvm.ScummVM</id>
```

Comment:
By the way, we should probably consider renaming this Flatpak to `org.scummvm.scummvm` or changing the upstream ID back to an uppercase one.